### ISSUE TITLE
Disabled days

### ISSUE THREAD

**Issue by filippos13** [hernansartorio/react-nice-dates#76]
How can I disable specific days? For example the 1st, 23rd and 25th of a month?
Is there any way I can disable specific days using an date array?
Thank you

**Comment by ITernovtsii** [comment]
yes, check modifiers docs https://reactnicedates.hernansartorio.com/#api-reference
something like this will work
```
modifiers={{
  disabled: (date: Date) => [1, 23, 25].includes(date.getDate()),
}}
```
